draft/2024-07-31-this-week-in-rust.md

```diff
@@ -87,6 +87,8 @@ If you are a Rust project owner and are looking for contributors, please submit
 
 [guidelines]:https://github.com/rust-lang/this-week-in-rust?tab=readme-ov-file#call-for-participation-guidelines
 
+Hi, I have a few OSS projects in need for contributors. More details [here](https://users.rust-lang.org/t/looking-for-contributors-to-open-source-projects/115191).
+
```

Comment:
Consider reading the guidelines to be included https://github.com/rust-lang/this-week-in-rust?tab=readme-ov-file#call-for-participation-guidelines

Reply:
sorry bit I did, can you please tell me what I'm doing wrong

Reply:
Hello! Ty for submitting this :) 

The format we prefer is links to individual issues on a project, [you can see a good example here ](https://github.com/rust-lang/this-week-in-rust/blob/621ded70e552893cb7ae126004c0ee4bb09f04c0/content/2024-01-03-this-week-in-rust.md?plain=1#L81)

Do you mind picking out a few issues for a few of those projects and submitting those to next week's issue?

Thank you!

Reply:
> Hello! Ty for submitting this :)
> 
> The format we prefer is links to individual issues on a project, [you can see a good example here ](https://github.com/rust-lang/this-week-in-rust/blob/621ded70e552893cb7ae126004c0ee4bb09f04c0/content/2024-01-03-this-week-in-rust.md?plain=1#L81)
> 
> Do you mind picking out a few issues for a few of those projects and submitting those to next week's issue?
> 
> Thank you!

thank you for your response, smth like this?
https://github.com/radumarias/rencfs/issues?q=is%3Aissue+label%3Afirst-timers-only+label%3A%22good+first+issue%22+is%3Aopen

or create a list of links with all of them? it's ok to post here or where exactly?

Reply:
I would pick out just a few specific issues at first (maybe 2-3 for up to 4 projects), and then we can include more in future issues. Could you open a new pull request against the current draft?

Reply:
[This is the current draft](https://github.com/rust-lang/this-week-in-rust/blob/master/draft/2024-08-07-this-week-in-rust.md)

Reply:
Thank you, I created this https://github.com/rust-lang/this-week-in-rust/pull/5790